gcdsp.py

```diff
@@ -86,36 +54,36 @@ def parse(self, res, byte1, byte2):
 
         if type == OpType.REG:
             res.type = o_reg
-            res.dtyp = dt_byte  # TODO: fix (ACCs are 40-bit for example)
+            res.dtype = dt_byte  # TODO: fix (ACCs are 40-bit for example)
             res.reg = val
         elif type == OpType.PRG:
             res.type = o_phrase
-            res.dtyp = dt_byte
+            res.dtype = dt_byte
             res.phrase = val
         elif type == OpType.ADDR_I:
             res.type = o_near
-            res.dtyp = dt_byte
+            res.dtype = dt_byte
             res.addr = val
         elif type == OpType.ADDR_D:
             res.type = o_mem
-            res.dtyp = dt_byte
+            res.dtype = dt_byte
             res.addr = val
         elif type == OpType.IMM:
             res.type = o_imm
-            res.dtyp = dt_byte
+            res.dtype = dt_byte
             res.value = val
         elif type == OpType.MEM:
             if self.size != 2:
                 b = val & 0x80
                 if b:
                     val |= 0xFF00
             res.type = o_mem
-            res.dtyp = dt_byte
-            res.addr = val
+            res.dtype = dt_byte
+            res.addr = 0x10000 | val
```

Comment:
Please add to the readme how the segments should be set up so that this works. (For a future PR, could the plugin do that itself?)